kubectl-intermediate/step6.md

```diff
@@ -1,17 +1,12 @@
 Now that we have created two new namespaces, we can deploy a container in each of the namespaces.
 
-Let's  deploy a single `redis` container into the `backend` namespace using  `kubectl run` with the `--restart=Never`. Type the below command:
+Let's  deploy a single `redis` container into the `backend` namespace using `kubectl run`. Type the below command:
 
-`kubectl run redis --image=redis -n backend --restart=Never `{{execute}}
```

Comment:
Why the removal of `--restart=Never`. I'm not aware of the history behind why it was added in the original version of the course.

Reply:
afaict the usage of --restart=Never used to be a common practice with earlier versions of k8s. the main reason you see it removed is that, afaict, it serves no purpose.

Reply:
OK so it was not needed for the workshop to function. Thanks for confirming.